ui/src/components/auth/RecoverPassword.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import axios from 'axios';
+import {
+  Box,
+  Button,
+  Stack,
+  TextField,
+  Typography,
+  Snackbar,
+  Alert,
+  AlertColor,
+} from '@mui/material';
+import { Link as RouterLink } from 'react-router-dom';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { z } from 'zod';
+import { RecoverPasswordFields } from './helpers/form.types';
+import { formSpacing, formWidth } from './helpers/constants';
+import { ArrowBackRounded } from '@mui/icons-material';
+import { emailValidator } from './helpers/zod.validators';
+
+// zod validation
+const recoverPwdValidationSchema = z.object({
+  email: emailValidator,
+});
+
+const RecoverPasswordContainer: React.FC = (props: any) => {
```

Comment:
Props is unused? 

I think theres a convention abt naming props type. Like u hav to name it filename + props, loke here its 

`type RecoverPasswordContainerProps = {}`
Better use this as placeholder for other file props too ^